src/System.Runtime/tests/System/StringTests.cs

```diff
@@ -2521,9 +2702,33 @@ public static void ToUpper_TurkishI_TurkishCulture()
             RemoteInvoke(() =>
             {
                 CultureInfo.CurrentCulture = new CultureInfo("tr-TR");
-                Assert.True("H\u0069 World".ToUpper().Equals("H\u0130 WORLD", StringComparison.Ordinal));
```

Comment:
it would be a good idea to convert this method to be Theory and pass the data instead of adding the test string inline in the code

Reply:
this apply for the next 2 methods too.

---
In reply to: [176479037](https://github.com/dotnet/corefx/pull/28347#discussion_r176479037) [](ancestors = 176479037)